content/articles/understanding-the-51-attack-on-blockchain/index.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /understanding-the-51-attack-on-blockchain/
+title: Understanding the 51% attack on blockchain
+description: This article will cover the 51% attack on the blockchain. The reader will understand what it entails, its risks, and possible ways to prevent it.
+author: ephraim-njoroge
+date: 2021-12-08T00:00:00-17:00
+topics: [Blockchain]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/understanding-the-51-attack-on-blockchain/hero.jpeg
+    alt: Understanding the 51% attack on blockchain example Image
+---
+[Blockchain](https://www.ibm.com/topics/what-is-blockchain) technology is not a new term in today’s digital world. The technology has existed over the past few years. 
+<!--more-->
+It is prevalent when cryptocurrencies such as [Bitcoin](https://bitcoin.org/en/) are mentioned. However, according to numerous research reports, businesses have not fully adopted the use of cryptocurrencies. Most of these businesses are relying on traditional methods in their operations instead of using blockchain technology.
+
+Blockchain technology can be described as a structure that stores public transaction records in various databases in a network-connected environment through peer-to-peer nodes. The transactional records are known as blocks, and the various interconnected databases are referred to as a chain.
+
+Each transaction in a blockchain is authorized using the owner's digital signature, which safeguards it from tampering. Due to this, the information is highly secured. Blockchain technology utilizes the hash encryption algorithms, in most cases [SHA256](https://www.n-able.com/blog/sha-256-encryption), to ensure that data is secure.
+
+Blockchain technology involves adding transaction details to digital ledgers through a process known as mining. Mining involves generating the hash for the block of the transaction, which ensures that blockchain is secure.
+
+This article will cover the 51% attack on the blockchain. The reader will understand what it entails, its risks, and possible ways to prevent it.
+
+### Overview
+1. [Introduction](#introduction)
+2. [Understanding 51% attack on Bitcoin and Blockchain](#understanding-51%-attack-on-bitcoin-and-blockchain)
+3. [The impacts of 51% attack on Blockchain and Bitcoin](#the-impacts-of-51%-attack-on-blockchain-and-bitcoin)
+4. [Risks involved with 51% attack](#risks-involved-with-51%-attack)
+4. [The platforms faced a 51% attack](#the-platforms-faced-a-51%-attack)
+5. [Comparing a 51% attack with a 34% attack](#comparing-a-51%-attack-with-a-34%-attack)
+6. [Ways to prevent a 51% attack](#ways-to-prevent-a-51%-attack)
+7. [Can 51% attack reoccur?](#can-51%-attack-reoccur?)
```

Comment:
Inside the section in brackets which begins with a hash sign, remove the % sign for it to be rendered.